src/lib/bootloader/systemdboot.rb

```diff
@@ -0,0 +1,304 @@
+# frozen_string_literal: true
+
+require "fileutils"
+require "yast"
+require "bootloader/sysconfig"
+require "bootloader/cpu_mitigations"
+
+Yast.import "Report"
+Yast.import "Arch"
+Yast.import "ProductFeatures"
+Yast.import "BootStorage"
+Yast.import "Stage"
+
+module Bootloader
+  # Represents systemd bootloader with efi target
+  # rubocop:disable Metrics/ClassLength
+  class SystemdBoot < BootloaderBase
+    include Yast::Logger
+    include Yast::I18n
+
+    # @!attribute menue_timeout
+    #   @return [Integer] menue timeout
+    attr_accessor :menue_timeout
+
+    # @!attribute secure_boot
+    #   @return [Boolean] current secure boot setting
+    attr_accessor :secure_boot
+
+    def initialize
+      super
+
+      textdomain "bootloader"
+      self.menue_timeout = Yast::ProductFeatures.GetIntegerFeature("globals", "boot_timeout").to_i
+    end
+
+    def merge(other)
+      log.info "merging with system: timeout=#{other.menue_timeout} " \
+               "secure_boot=#{other.secure_boot}"
+      super
+      self.menue_timeout = other.menue_timeout unless other.menue_timeout.nil?
+      self.secure_boot = other.secure_boot unless other.secure_boot.nil?
+    end
+
+    def cpu_mitigations
+      Bootloader::CpuMitigations.new(:manual)
+    end
+
+    def cpu_mitigations=(value)
+      log.info "ignoring set of cpu_mitigations (#{value}) in systemd bootloader."
+    end
```

Comment:
as here it is ignored. Do we even need this methods? It should behave same as for "none" bootloader and basically do not show cpu mitigation when it is not written.

Reply:
I have done it like "none" bootloader. So, I have removed it.